homeassistant/components/webostv/quality_scale.yaml

```diff
@@ -0,0 +1,86 @@
+rules:
+  # Bronze
+  action-setup:
+    status: todo
+    comment: move actions to entity services
+  appropriate-polling: done
+  brands: done
+  common-modules:
+    status: exempt
+    comment: The integration does not use common patterns.
+  config-flow-test-coverage: todo
+  config-flow:
+    status: todo
+    comment: test_form - remove duplicated async_init, make sure tests ends with CREATE_ENTRY or ABORT, remove legacy uuid migration, remove name parameter, use hass.config_entries.async_setup instead of async_setup_component, snapshot in diagnostics (and other tests when possible), test_client_disconnected validate no error in log, make reauth flow more graceful
```

Comment:
```suggestion
    comment: remove duplicated config flow start in tests, make sure tests ends with CREATE_ENTRY or ABORT, remove legacy uuid migration, remove name parameter, use hass.config_entries.async_setup instead of async_setup_component, snapshot in diagnostics (and other tests when possible), test_client_disconnected validate no error in log, make reauth flow more graceful
```